decidim-blogs/app/views/decidim/blogs/posts/show.html.erb

```diff
@@ -15,47 +15,33 @@
   )
 %>
 
-<div class="row column view-header">
-  <div class="m-bottom">
-    <%= link_to :posts, class: "small hollow" do %>
-      <%= icon "chevron-left", class: "icon--small", role: "img", "aria-hidden": true %>
-      <%= t(".back") %>
-    <% end %>
-  </div>
-  <h2 class="heading2"><%= translated_attribute post.title %></h2>
-
-  <% post_presenter = Decidim::Blogs::PostPresenter.new(post) %>
-  <%= cell "decidim/author", post_presenter.author, from: post %>
-</div>
-<div class="row">
-  <% if show_endorsements_card? %>
-      <div class="columns section view-side mediumlarge-4 mediumlarge-push-8 large-3 large-push-9">
-        <div class="card">
-          <div class="card__content">
-            <div class="row collapse buttons__row">
-              <% if endorsements_enabled? %>
-                <div class="column small-9 collapse">
-                  <%= endorsement_buttons_cell(post) %>
-                </div>
-              <% end %>
-              <div class="column collapse <%= endorsements_enabled? ? "small-3" : "" %>">
-                <%= link_to "#comments", class: "button small compact hollow button--nomargin expanded" do %>
-                    <%= icon "comment-square", class: "icon--small", aria_label: t(".comments"), role: "img" %> <%= post.comments_count %>
-                <% end %>
-              </div>
-            </div>
-            <br>
-            <%= follow_button_for(post) %>
-          </div>
-        </div>
-      </div>
+<% content_for :css_content do %>
+  <%= stylesheet_pack_tag "decidim_blogs", media: "all" %>
+<% end %>
+
+<%= render layout: "layouts/decidim/shared/layout_center" do %>
```

Comment:
One general issue I still noticed is that there is not back link on the single blog post page to get back to the posts listing. I understand the redesign introduces the breadcrumbs navigation that would help with this but it would still be important to always have the back link to the listing page to make it easier to navigate the site.

Reply:
As you say, the idea is to use breadcrumb navigation to move between sections, but there are sections where the link may be necessary. We'll check it out later